src/iterateJsdoc.js

```diff
@@ -9,11 +9,21 @@ const parseComment = (commentNode, indent) => {
     // @see https://github.com/yavorskiy/comment-parser/issues/21
     parsers: [
       commentParser.PARSERS.parse_tag,
-      commentParser.PARSERS.parse_type,
+      (str, data) => {
+        if (data.tag === 'see') {
+          // @see can't contain types, only names or links which might be confused with types
+          return null;
```

Comment:
Ah, interesting. But we might want to see how jsdoc actually *behaves*. If you put in a type it may generate a link to it, despite what the docs say. The docs do refer to it referring to another "symbol" so it may be that a type is also included here.

Reply:
In that case perhaps it's better to be more lenient and completely ignore `@see {@link }` form at least until there's a piece of code that handles it specifically, this way types here will be treated as namepaths and validated the same.